pkg/cmd/grafana-cli/commands/mergefs_test.go

```diff
@@ -4,41 +4,65 @@ import (
 	"io/fs"
 	"os"
 	"path/filepath"
-	"strings"
 	"testing"
 	"testing/fstest"
 
 	"github.com/stretchr/testify/require"
 )
 
 func TestMergeFS(t *testing.T) {
+	var filePaths = []struct {
+		path           string
+		dirArrayLength int
+		dirs           []string
+	}{
+		// MapFS takes in account the current directory in addition to all included directories and produces a "" dir
+		{"a", 2, []string{"", "z"}},
+		{"a/z", 2, []string{"", "bar.cue"}},
+		{"b", 1, []string{"z"}},
+		{"b/z", 1, []string{"foo.cue"}},
+	}
+
 	tempDir := os.DirFS(filepath.Join("testdata", "mergefs"))
 	a := fstest.MapFS{
-		"a/":          &fstest.MapFile{Data: []byte("a")},
-		"a/z/":        &fstest.MapFile{Data: []byte("a/z")},
-		"a/z/foo.cue": &fstest.MapFile{Data: []byte("foo")},
+		"a/":          &fstest.MapFile{},
```

Comment:
Awkward, but without / it doesn't recognise it as a dir - only way I could make it work 🤷